<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValveMxSeries2Window</class>
 <widget class="QDialog" name="ValveMxSeries2Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Double 3-ways valve</string>
  </property>
  <widget class="QGroupBox" name="groupBox_valve2x3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Double 3-ways Valve</string>
   </property>
   <widget class="QRadioButton" name="radioButton_from_standards">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>From Standards (P1)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_to_chamber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>To Chambers (P2)</string>
    </property>
   </widget>
  </widget>
  <action name="action_pump">
   <property name="text">
    <string>Syringe pump</string>
   </property>
  </action>
  <action name="action_flows">
   <property name="text">
    <string>Flow controllers</string>
   </property>
  </action>
  <action name="action_pressure">
   <property name="text">
    <string>Pressure controller</string>
   </property>
  </action>
  <action name="action_valve2x3">
   <property name="text">
    <string>Double 3-ways valve</string>
   </property>
  </action>
  <action name="action_electrovalves">
   <property name="text">
    <string>Electro-valves</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
